src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -126,26 +126,41 @@ The .NET Foundation licenses this file to you under the MIT license.
 
   <!-- The properties below need to be defined only after we've found the correct runtime package reference -->
   <Target Name="SetupProperties" DependsOnTargets="$(IlcSetupPropertiesDependsOn)" BeforeTargets="Publish">
+    <ItemGroup>
+      <_NETCoreAppFrameworkReference Include="@(ResolvedFrameworkReference)" Condition="'%(ResolvedFrameworkReference.RuntimePackName)' == 'Microsoft.NETCore.App.Runtime.NativeAOT.$(RuntimeIdentifier)'" />
+    </ItemGroup>
+
     <PropertyGroup>
       <!-- Define paths used in build targets to point to the runtime-specific ILCompiler implementation -->
       <IlcToolsPath Condition="'$(IlcToolsPath)' == ''">$(IlcHostPackagePath)\tools\</IlcToolsPath>
       <IlcSdkPath Condition="'$(IlcSdkPath)' == ''">$(RuntimePackagePath)\sdk\</IlcSdkPath>
```

Comment:
Do we really need both IlcSdkPath and IlcPrivateSdkPath? Should we just rename IlcSdkPath to IlcPrivateSdkPath everywhere?

Reply:
If we really need both, we can use a comment that explains the difference between the two.

Reply:
It was written under the assumption that we are still referencing something from the `sdk` directory of the host ILCompiler NuGet. I double checked it now and we don't actually do that.

Given the fact that there are some overrides of `IlcSdkPath` for live builds I suppose it would make sense to keep the original name for now.

Reply:
Fix: https://github.com/dotnet/runtime/commit/efa7e2ffcbe30e1188213b7d62ba7feed45c2ead